pkg/vault/contracts/VaultCommon.sol

```diff
@@ -104,17 +104,15 @@ abstract contract VaultCommon is IVaultEvents, IVaultErrors, VaultStorage, Reent
         // Calculate the new delta after accounting for the change.
         int256 next = current + delta;
 
-        unchecked {
```

Comment:
hm, good point.

This is from the time we were not using `tIncrement` and `tDecrement`.